unlock-app/src/components/creator/CreatorLockForm.js

```diff
@@ -29,59 +29,81 @@ import {
 
 import { INFINITY, UNLIMITED_KEYS_COUNT } from '../../constants'
 
+/**
+ * Converts the lock values into form values
+ * @param {*} lockValues
+ */
+export const lockToFormValues = lockValues => {
+  const formValues = Object.assign({}, lockValues)
+
+  // In the form, duration is shown in days, vs seconds in the lock object
+  formValues.expirationDuration =
+    lockValues.expirationDuration / lockValues.expirationDurationUnit
+
+  // Unlimited keys
+  if (lockValues.maxNumberOfKeys === UNLIMITED_KEYS_COUNT) {
+    formValues.unlimitedKeys = true
+    formValues.maxNumberOfKeys = INFINITY
+  }
+
+  return formValues
+}
+
+/**
+ * Converts the form values into lock values
+ * @param {*} lockValues
+ */
+export const formValuesToLock = formValues => {
+  const lockValues = {}
+
+  lockValues.keyPrice = formValues.keyPrice
+  lockValues.maxNumberOfKeys = formValues.maxNumberOfKeys
+  lockValues.name = formValues.name
+  lockValues.address = formValues.address
+
+  // In the form, duration is shown in days, vs seconds in the lock object
+  lockValues.expirationDuration =
+    formValues.expirationDuration * formValues.expirationDurationUnit
+
+  // Unlimited keys
+  if (formValues.unlimitedKeys) {
+    lockValues.maxNumberOfKeys = UNLIMITED_KEYS_COUNT
```

Comment:
this conversion should also be done in walletService.createLock, the idea being that in unlock-app, infinite keys is represented on the lock object, and the conversion to chain-specific representation is in the services.

Reply:
so... I think this is a different issue. The goal of this PR is to make sure that the form component's input and output are consistent with the data format used by the unlock-app. 
Another PR can/will isolate the walletService logic/format from the unlock-app logic/format in the same way.